project/Dependencies.scala

```diff
@@ -21,12 +21,12 @@ object Dependencies {
   val workbenchUtilV = "0.5-4c7acd5"
   val workbenchModelV = "0.14-2e155f0"
   val workbenchGoogleV = "0.21-64a7b29"
-  val workbenchGoogle2V = "0.13-6f4d8f1"
+  val workbenchGoogle2V = "0.13-9be71b5-SNAP"
```

Comment:
both the workbenchGoogle2V and the helmScalaSdkV  here are TODO when the respective PR is merged